go-controller/Makefile

```diff
@@ -12,14 +12,14 @@ GOPATH ?= $(shell go env GOPATH)
 TEST_REPORT_DIR?=$(CURDIR)/_artifacts
 export TEST_REPORT_DIR
 GO_VERSION ?= 1.24
-GO_DOCKER_IMG = quay.io/lib/golang:${GO_VERSION}
+GO_DOCKER_IMG = quay.io/projectquay/golang:${GO_VERSION}
```

Comment:
I'm assuming this image is well maintained given it's from projectquay - but I'm not really sure.